src/admin/header/Header.tsx

```
@@ -0,0 +1,146 @@
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import HelpIcon from '../dashboard/icons/HelpIcon';
+import CrownIcon from '../dashboard/icons/CrownIcon';
+import IconMapping from '../dashboard/components/IconMapping';
+import { twMerge } from 'tailwind-merge';
+
+const Header = () => {
+    const [ showDropdown, setShowDropdown ] = useState( false );
+    const { is_pro_exists, lite_version, help_menu_items, has_new_version } =
+        dokanAdminPanelHeaderSettings?.header_info;
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)